app/views/handlers/individual_response.py

```diff
@@ -121,7 +121,8 @@ def __init__(
         self._form_data = form_data
         self._answers: dict[str, Any] | None = None
         self._list_item_id = list_item_id
-        self._list_name = self._schema.get_individual_response_list()
+        # Type ignore: in individual_response for_list is required which is where list_name comes from
```

Comment:
I have tried changing the code but the unit tests are failing which would mean more of an investigation into the tests/code which would be time consuming and not as critical of an issue considering individual_response isn't used.

Will leave as errors for now so that they do not get missed.

Reply:
I think this one is controversial - `get_individual_response_list` is a method on `questionnaire_schema` which can indeed return None, for most schemas, however its only ever used here where it can't. I think the method ought to move, but I don't think its in the scope of the ticket, so will resolve based on it being a largely unused file and it leaves the typing in `questionnaire_schema` technically more correct... I think

Reply:
I think it's worth leaving as is since the scope of this ticket would grow further - my suggestion is resolve this comment, but make a ticket to investigate it as it's own issue. Should hopefully be a small one to pick up at a later date.